Netkan/CmdLineOptions.cs

```diff
@@ -49,11 +49,7 @@ internal class CmdLineOptions
         [Option("validate-ckan", HelpText = "Name of .ckan file to check for errors")]
         public string ValidateCkan { get; set; }
 
-        [Option("version", HelpText = "Display the netkan version number and exit")]
-        public bool Version { get; set; }
-
-        // TODO: How do we mark this as required?
-        [ValueOption(0)]
+        [Value(0, Required = true, HelpText = "Name of .ckan file to process.")]
```

Comment:
Will setting this to required break the bot? It runs it just with `--queues`, `--github-token`, and `--cachedir`.

https://github.com/KSP-CKAN/CKAN/blob/ba54c7972d05f6abe8efcbc787122a39acf0e6ef/Dockerfile.netkan#L7-L8